list_app.py

```diff
@@ -0,0 +1,12 @@
+cars = []
+num = int(input("How many cars do you want to add?\n"))
+
+while num != 0:
+	car = input("Type your car\n")
+	cars.append(car)
+	num -= 1
```

Comment:
One quick suggestion here :) I have noticed usage of `tab`(8 spaces in your editor) here and in the other lines! But, I think the general convention in python community is that we probably should use `4 spaces`. What do you think?

```suggestion
    car = input("Type your car\n")
    cars.append(car)
    num -= 1
```